plugins/modules/mysql_user.py

```diff
@@ -22,7 +22,7 @@
     required: true
   password:
     description:
-      - Set the user's password.
+      - Set the user's password. Only for C(mysql_native_password) authentication. For other authentication plugins see the combination of I(plugin), I(plugin_hash_string), I(plugin_auth_string).
```

Comment:
```suggestion
      - Set the user's password. Only for C(mysql_native_password) authentication.
        For other authentication plugins see the combination of I(plugin), I(plugin_hash_string), I(plugin_auth_string).
```
it should fix `ERROR: plugins/modules/mysql_user.py:25:161: E501: line too long (195 > 160 characters)` sanity error